messaging/topic_mgt.go

```diff
@@ -20,14 +20,17 @@ import (
 	"fmt"
 	"net/http"
 	"strings"
+	"time"
 
 	"firebase.google.com/go/internal"
 )
 
 const (
-	iidEndpoint    = "https://iid.googleapis.com/iid/v1"
-	iidSubscribe   = "batchAdd"
-	iidUnsubscribe = "batchRemove"
+	iidBaseEndpoint = "https://iid.googleapis.com/iid"
```

Comment:
Note:
We are in the process of deprecating everything related to Firebase Instance ID in the year 2020.
All Topic related features will move into Firebase Messaging and most likely the endpoint(s)s will move to be an FCM domain.
Thus, I recommend not to add new client logic that depends on a server endpoint that we plan to deprecate soon.

Maybe, we can the same endpoints on the FCM domain `fcm.googleapis.com` and use those URLs instead.
Probably we can just add a URL `/topics` to the existing GFE config file of `fcm.googleapis.com` and point it to the same IID logic (server-side).
That way we could release new client software with that URL and without using endpoints that are about to be deprecated.

Reply:
Follow up on andirayo@'s comment: 
The timeline of moving to endpoints under fcm.googleapis.com is not determined. We decide to not block this PR. 
Please feel free to use iid.googleapis.com in the PR.

Reply:
+1